gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorage.java

```diff
@@ -162,6 +173,18 @@ default SeekableByteChannel open(StorageResourceId resourceId) throws IOExceptio
   SeekableByteChannel open(StorageResourceId resourceId, GoogleCloudStorageReadOptions readOptions)
       throws IOException;
 
+  /**
+   * Opens an object for reading using GoogleCloudStorageItemInfo
+   *
+   * @param itemInfo
+   * @param readOptions
+   * @return a channel for reading from the given object
+   * @throws java.io.FileNotFoundException if the given object does not exist
+   * @throws IOException if object exists but cannot be opened
+   */
+  SeekableByteChannel open(
```

Comment:
To ensure backwards compatibility, can we add a `default` impl for the new methods here

Reply:
I will add a default implementation for the newly added method